Trinap/Sources/Data/Repositories/User/DefaultUserRepository.swift

```diff
@@ -84,4 +83,13 @@ final class DefaultUserRepository: UserRepository {
             .updateDocument(collection: .users, document: userId, values: values)
             .asObservable()
     }
+    
+    func createRandomNickname() -> Observable<String> {
+        let endpoint = RandomNicknameEndpoint.main
+        return self.networkService.request(endpoint)
+            .compactMap { data in
+                let nickName = try? JSONDecoder().decode(NicknameDTO.self, from: data)
+                return nickName?.words.first
+            }
```

Comment:
rx의 `.decode` operator 사용해도 좋을 것 같습니다!

```suggestion
        return self.networkService.request(endpoint)
            .decode(type: NicknameDTO.self, decoder: JSONDecoder())
            .compactMap { $0.words.first }
```